README.md

```diff
@@ -208,55 +167,170 @@ Containers are also available for each git commit to the `main` branch under `${
 
 [Docker]: https://docs.docker.com/get-docker/
 [Chainguard Images]: https://github.com/chainguard-images/images
+
+### Write your own schema and relationships
+
+The first step to using SpiceDB is to define your **schema** and write **relationships** that represent the permissions in your application. There are various way to do this:
+
+- Use the [client libraries]
+- Run the open-source [Playground] locally. You can also use the [hosted Playground]; it contains a set of example schemas that you can test.
```

Comment:
Switch: `Use the [hosted Playground] or [run it yourself locally]. The playground contains a set of example schemas and test data for different scenarios`